plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveMetadata.java

```diff
@@ -73,6 +74,25 @@ public void testCreatePredicate()
                 false));
     }
 
+    @Test
+    public void testCreatePredicateWithEmptyPartition()
+    {
+        ImmutableList.Builder<HivePartition> partitions = ImmutableList.builder();
+        Domain domain = createPredicate(ImmutableList.of(TEST_COLUMN_HANDLE), partitions.build())
+                .getDomains().orElseThrow().get(TEST_COLUMN_HANDLE);
+        assertEquals(domain, Domain.all(VARCHAR));
+    }
+
+    @Test
+    public void testCreatePredicateWithEmptyPartitionAndEmptyColumn()
+    {
+        ImmutableList.Builder<HivePartition> partitions = ImmutableList.builder();
+        ImmutableList.Builder<ColumnHandle> columnHandle = ImmutableList.builder();
+        Domain domain = createPredicate(columnHandle.build(), partitions.build())
```

Comment:
Replace with:
`createPredicate(ImmutableList.of(), ImmutableList.of()).getDomains().orElseThrow().get(TEST_COLUMN_HANDLE)`

Reply:
fixed